### PR TITLE
Allow using the Rally.js support library without a bundler.

### PR DESCRIPTION
**Pull request by Dexterp37**
This is preparatory work to use rally.js within integration tests.

TODO:

- [x] Perform a manual QA round.

Checklist for reviewer:

- [ ] The description should reference a bug or github issue, if relevant.
- [ ] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
- [ ] Any version increase must follow the [release process](./docs/RELEASE_PROCESS.md).

### CODE REVIEW

**Comment by Dexterp37** [comment]
I tested this manually, everything works just fine. This is good to review @rhelmer !

File: support/rally.js [hunk 1]
```diff
@@ -179,3 +179,8 @@ module.exports = class Rally {
     }
   }
 }
+
+// Make this library Require-able.
```
Comment (brizental) [inline support/rally.js:183]: I believe you can change your output format from `iife` to `umd` and then you don't need to do this.

**Review by brizental — APPROVED** [review]
r+wc

File: support/rally.js (hunk 1 above)
Comment (Dexterp37) [reply support/rally.js:183]: Thanks, as we talked on Matrix we won't be always using a bundler (e.g. tests) so that won't help :(

**Pull request merged**